background/examplebg.js

```diff
@@ -0,0 +1,20 @@
+const slug = "examplebg";
+
+module.exports.conf = {
+  slug: slug,
+  interval: 3000 // milliseconds
+};
+
+module.exports.init = () => {
+  console.log("Worker has been initialized.");
+};
+
+var ranTimes = 0;
+
+module.exports.run = (client) => {
+  console.log(`${slug} worker ran ${++ranTimes} times.`);
+  if (ranTimes == 4) {
+    console.log(`${slug} ran 4 times. Unloading..`);
+    client.unloadBackgroundWorker(slug);
+  }
+}
```

Comment:
Semicolon necessary here, this will fix the Codacy issue also

Reply:
The init() function seems useless too. What's the point of calling another init function if you're just console logging stuff? Seems redundant.

Reply:
He is console.logging stuff in an example background worker file, however you can use the init function to do more than that if i understood correctly

Reply:
Plus the init runs once/on every load, and run runs on the interval